src/procedures/the-cat-api/the-cat-api.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * The Cats Service provides access to the Cat API data from https://api.thecatapi.com/v1/.
+ *
+ * @service
+ * @category Society
```

Comment:
`@category Media` would also be good since this returns images. You can do multiple categories by just putting them on different lines. The service will be placed in all of its categories.